### PR TITLE
configure.ac: add --enable-deprecated flag

### PR DESCRIPTION
**Pull request by brauner**

### CODE REVIEW

File: doc/Makefile.am [hunk 1]
```diff
@@ -50,8 +49,10 @@ man_MANS = \
 	\
 	lxc.7
 
+if ENABLE_DEPRECATED
 if ENABLE_PYTHON
-    man_MANS += lxc-start-ephemeral.1
+    man_MANS += lxc-start-ephemeral.1 lxc-clone.1
```
Comment (stgraber) [inline doc/Makefile.am]: lxc-clone shouldn't be under ENABLE_PYTHON

File: doc/ja/Makefile.am [hunk 2]
```diff
@@ -42,8 +41,10 @@ man_MANS = \
 	\
 	lxc.7
 
+if ENABLE_DEPRECATED
 if ENABLE_PYTHON
-    man_MANS += lxc-start-ephemeral.1
+    man_MANS += lxc-start-ephemeral.1 lxc-clone.1
```
Comment (stgraber) [inline doc/ja/Makefile.am]: lxc-clone shouldn't be under ENABLE_PYTHON

File: doc/ko/Makefile.am [hunk 3]
```diff
@@ -42,8 +41,10 @@ man_MANS = \
 	\
 	lxc.7
 
+if ENABLE_DEPRECATED
 if ENABLE_PYTHON
-    man_MANS += lxc-start-ephemeral.1
+    man_MANS += lxc-start-ephemeral.1 lxc-clone.1
```
Comment (stgraber) [inline doc/ko/Makefile.am]: lxc-clone shouldn't be under ENABLE_PYTHON

File: src/lxc/lxc-start-ephemeral.in [hunk 4]
```diff
@@ -36,6 +36,10 @@ import tempfile
 _ = gettext.gettext
 gettext.textdomain("lxc-start-ephemeral")
 
+def printstderr(*args):
+    print("lxc-start-ephemeral is deprecated in favor of lxc-copy\n", *args, file = sys.stderr)
```
Comment (stgraber) [inline src/lxc/lxc-start-ephemeral.in]: Pretty sure pep8 would be unhappy with that "file = sys.stderr", should be "file=sys.stderr".

For python changes, it's usually a good idea to make sure that the code is clean when running "pep8" and "pyflakes3" on it.

File: doc/Makefile.am (hunk 1 above)
Comment (brauner) [inline doc/Makefile.am]: addressed.

File: doc/ja/Makefile.am (hunk 2 above)
Comment (brauner) [inline doc/ja/Makefile.am]: addressed.

File: doc/ko/Makefile.am (hunk 3 above)
Comment (brauner) [inline doc/ko/Makefile.am]: addressed.

File: src/lxc/lxc-start-ephemeral.in (hunk 4 above)
Comment (brauner) [inline src/lxc/lxc-start-ephemeral.in]: adressed. Checked with pep8 and pyflakes3k. Thanks!

**Comment by brauner** [comment]
jenkins: test this please

**Pull request merged**